### PR TITLE
Added docs for Stacks CLI

### PR DESCRIPTION
**Pull request by russellseymour**
#### 📲 What

Added documentation from the builds of Stacks CLI to the website
Implements [AB#3951](https://amido-dev.visualstudio.com/73884c9a-a68f-4f67-b2b5-b588c2eb8492/_workitems/edit/3951)

#### 🤔 Why
		
The Stacks CLI has been re-written in Golang and has a new interface. The documentation shows how this CLI is to be used with examples.
		
#### 🛠 How
		
Documentation is built by the CLI. The outputs from that are added to this repository for the Stacks CLI.

#### 👀 Evidence

Following screenshot shows the docs running in a Docker container on local machine.
		
image

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/stackscli/runtime_config.mdx [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+| -------------------------- | ---------------------------------- | ----------------------------------------------------------------------------- | --------------------- | -------------------------------------------- |
+| `-c`, `--config`           | AMIDOSTACKS\_CONFIG                | Path to the configuration file to use                                         |                       |                                              |
+| `-l`, `--loglevel`         | AMIDOSTACKS\_LOG\_LEVEL            | Level of logging to be used                                                   | `info`                | `debug`, `info`, `warning`, `error`, `fatal` |
+| `-f`, `--logformat`        | AMIDOSTACKS\_LOG\_FORMAT           | Output type of the logging                                                    | `text`                | `text`, `json`                               |
+| `--logcolur`               | AMIDOSTACKS\_LOG\_COLOUR           | State if logging should include colour                                        |                       |                                              |
```
Comment (amwis) [inline docs/stackscli/runtime_config.mdx:26]: Potential typo here, it should be `--logcolour`, right?

Comment (russellseymour) [reply docs/stackscli/runtime_config.mdx]: Corrected the typo

**Review by amwis — APPROVED** [review]

**Pull request merged**